webrender/src/render_task.rs

```diff
@@ -514,6 +524,26 @@ impl RenderTask {
         }
     }
 
+    // Check if this task wants to be made available as an input
+    // to all passes in the render task tree. To qualify for this,
```

Comment:
correction: all passes *except* the first one

Reply:
Fixed